api/models/linux/namespace/namespace.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package linux.namespace;
+
+option go_package = "github.com/ligato/vpp-agent/api/models/linux/namespace;linux_namespace";
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+option (gogoproto.messagename_all) = true;
+
+message NetNamespace {
+    enum ReferenceType {
+        UNDEFINED = 0;
+        NSID = 1;         /* named namespace */
```

Comment:
When using NSID... how are you figuring out which directory to look for the NSID file for the named namespace?

Reply:
^^^^^^^

Reply:
This is done by the external package `vishvananda/netns`, which has hard-coded `/var/run/netns` directory:
https://github.com/ligato/vpp-agent/blob/dev/vendor/github.com/vishvananda/netns/netns_linux.go#L80